recipes/snver/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "snver" %}
+{% set version = "0.5.3" %}
+
+about:
+  home: http://snver.sourceforge.net/
+  license: GNU General Public License version 3.0 (GPLv3)
```

Comment:
GPL requires that distributions be accompanied by a copy of the license, so `license_file` must be set. Should be somewhere in the tarball.